src/App.vue

```diff
@@ -2,3 +2,23 @@
 <template>
   <RouterView />
 </template>
+
+<script setup>
+import useLocalStorage from 'src/compositionFunctions/useLocalStorage'
+import { useQuasar } from 'quasar'
+import { ref, onMounted } from 'vue'
+import { useRouter } from 'vue-router'
+const router = useRouter()
+const { retrieveUserData } = useLocalStorage()
+const { retrieveDarkMode } = useLocalStorage()
+const darkMode = ref(retrieveDarkMode())
+const $q = useQuasar()
+onMounted(() => {
+  if (darkMode.value) $q.dark.set(true)
+  if (retrieveUserData()) {
```

Comment:
It's better to call the function outside of IF and use its result for better readability